README.md

```diff
@@ -19,164 +20,141 @@ You should demonstrate an ability to:
 1. `cd` into the dir created `$ cd grocery-store`
 1. Run `git remote -v` to verify the folder you are in corresponds to the fork you have created.  
   If it is **correct** it will include your username
-  If it is **incorrect** it will include "AdaGold" or "Ada-C8" 
+  If it is **incorrect** it will include "AdaGold" or "Ada-C#"
 1. Run `gem install minitest-skip` to install an extra gem for testing (more on what this actually does later).
 
 ### Testing
 
-This is our first project with real tests! Following the instructions from the [TDD lecture](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/intro-to-automated-tests.md), there are three things in our project directory:
+This is the first project where you'll be writing your own tests. Following the instructions from the [TDD lecture](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/intro-to-automated-tests.md), there are three things in our project directory:
 
 ```
 Rakefile
 lib/
 specs/
 ```
 
-Each class you write (there will only be one until wave 3) should get its own file, `lib/class_name.rb`. The specs for that class will be in `specs/class_name_spec.rb`, and you can run all specs using the `rake` command from your terminal.
-
+Each class you write should get its own file, `lib/class_name.rb`. The specs for that class will be in `specs/class_name_spec.rb`, and you can run all specs using the `rake` command from your terminal.
 
 ## Wave 1
 
 ### Learning Goals
-- Create a **class** inside of a **module**
-- Create **methods** inside the **class** to perform actions
-- Learn how Ruby does error handling
+- Create a **class**
+- Write **instance methods** inside a **class** to perform actions
+- Link two classes using **composition**
+- Use exceptions to handle errors
 - Verify code correctness by **testing**
 
 ### Testing
 For Wave 1, all tests have been provided for you. For each piece of functionality that you build, you should run the tests from the command line using the `rake` command. To focus on only one test at a time, change all `it` methods to `xit` except for the **one test** you'd like to run. All tests provided should be passing at the end of your work on Wave 1.
 
 ### Requirements
 
-Create a `Grocery` module which will contain an `Order` class and any future grocery store logic.
+#### Customer
+
+Create a class called `Customer`. Each new Customer should include the following attributes:
+- ID, a number
+- Email address, a string
+- Delivery address, a hash
+
+ID should be _readable_ but not _writable_; the other two attributes can be both read and written.
+
+#### Order
 
 Create an `Order` class which should have the following functionality:
 - A new order should be created with:
```

Comment:
Instead of "A new order should be created with", I would mimic the language above in Customer in which you say "should include the following attributes." I think you could combine the two ideas, too, like "should include the following attributes, which will be passed in at the time of instantiation."

Also, I'm very tempted to suggest adding in the idea of "each of these attributes should be instance variables" explicitly, but...? Maybe that's unnecessary?